### ISSUE TITLE
Failed to featurize datapoint in Tutorial_Part_14_Modeling_Protein_Ligand_Interactions_with_Atomic_Convolutions.ipynb

### ISSUE THREAD

**Issue by andresilvapimentel** [deepchem/deepchem#2602]
## 🐛 Bug

In the command:
tasks, datasets, transformers = deepchem.molnet.load_pdbbind(featurizer=acf,
                                             save_dir='.',
                                             data_dir='.',
                                             pocket=True,
                                             reload=False,
                                             set_name='core')

Error: Failed to featurize datapoint 0. Appending empty array. repeated 192 times.
So, when I run the command line:
max_epochs = 50

for epoch in range(max_epochs):
  loss = acm.fit(train, nb_epoch=1, max_checkpoints_to_keep=1, all_losses=losses)
  metric = dc.metrics.Metric(dc.metrics.score_function.rms_score)
  val_losses.append(acm.evaluate(val, metrics=[metric])['rms_score']**2)  # L2 Loss

The error message "tuple index out of range" shows up.

How can I solve this error?

**Comment by rbharath** [comment]
Which version of DeepChem are you on? I recall this was an error we fixed a while back (you might need to update to DeepChem head since 2.6 isn't out yet)

**Comment by andresilvapimentel** [comment]
I am using 2.6.0.dev
Which version was this issue solved?

Em sex., 16 de jul. de 2021 às 13:21, Bharath Ramsundar <
***@***.***> escreveu:

**Comment by andresilvapimentel** [comment]
I tried to solve the issue  updating to DeepChem head, but it did not work
it.

Em sex., 16 de jul. de 2021 às 13:21, Bharath Ramsundar <
***@***.***> escreveu:

**Comment by rbharath** [comment]
I've gone ahead and marked this as a bug. If anyone else manages to reproduce, please make a note here. Let's see if we can figure this out...

**Comment by davidRFB** [comment]
Hi ! I was not available to reproduce exactly the error. However after add the line 
`!pip install rdkit-pypi`
before the import modules cell everything works well ! Maybe that help u

**Comment by andresilvapimentel** [comment]
Dear David, I had also done the same as you, but I was using a different
rdkit installation procedure that it was causing the issue.
Thanks for sharing your procedure.

Em ter, 3 de ago de 2021 01:39, davidRFB ***@***.***>
escreveu:
